sdk/main/src/impl/NodeAddress.cc

```diff
@@ -26,42 +28,159 @@
 
 namespace Hedera::internal
 {
+//-----
+NodeAddress::NodeAddress(const std::string ipAddressV4, const int port)
+{
+  std::vector<unsigned char> octets;
+  std::stringstream strStream(ipAddressV4);
+  std::string temp;
+
+  try
+  {
+    while (getline(strStream, temp, '.'))
+    {
+      int octet = atoi(temp.c_str());
+
+      if (octet > 0)
+      {
+        octets.push_back((unsigned char)octet);
```

Comment:
I'd recommend using C++ `static_cast<>` instead of C-style casting, as C-style casts can fail at runtime. Chances are this would work fine, but it's good to stay consistent across the codebase in my opinion.

Reply:
Updated in b5d9f2b66e3755cb32f28c33c8e5dff59922209a.